.github/workflows/masterbot.yml

```diff
@@ -0,0 +1,90 @@
+name: Master Robot
+on:
+  push:
+    branches:
+      - master
+    tags-ignore: ['*']
```

Comment:
The intent for this workflow is to run on all commits on `master` except tagged commits, which I assume to always be version bumps.